_topics/en/responsible-disclosures.md

```diff
@@ -36,6 +36,9 @@ optech_mentions:
   - title: "Trezor team disclosed a bug in the C-language bech32 specification affecting multiple wallets"
     url: /en/newsletters/2018/11/06/#overflow-bug-in-reference-c-language-bech32-implementation
 
+  - title: "Suhas Daftuar disclosed a bug that could temporarily exclude a Bitcoin Core node from consensus"
```

Comment:
This is not in reference to anything in this week's newsletter, but while reading Daftuar's amazing post on Delving this week, I was reminded of the also amazing description he wrote of merkle tree malleability problems years ago.  When I went looking for it, I realized (to my shame) I had never added it to our list of responsible disclosures.  This corrects that oversight, sorry Suhas!